crates/pyndakaas/src/lib.rs

```diff
@@ -636,6 +645,7 @@ mod pindakaas {
 
 		#[pyclass]
 		#[derive(Debug, Default)]
+		/// :meta private:
 		struct CaDiCaLInner(Mutex<Cadical>);
```

Comment:
Actually interesting that this is a `pyclass`, but also private. Should exported classes not be `pub`, just so that is a little more consistent, or is this more convenient in any way?

Reply:
Hmm.. I'm not exactly sure what the situation is here. So it's exported, but it should not? I'm not sure how to prevent the export of a pyclass inside a module which has `#[pymodule]`.